ci-operator/jobs/openshift/release/openshift-release-master-periodics.yaml

```diff
@@ -25625,8 +25625,6 @@ periodics:
     job-release: "4.9"
     pj-rehearse.openshift.io/can-be-rehearsed: "true"
   name: periodic-ci-openshift-release-master-ci-4.9-e2e-aws
-  prowjob_defaults:
-    tenant_id: gangway-api
```

Comment:
this probably isn't something we want to commit

Reply:
it was generated by `make jobs` it reverts once I rebase.